base/pgsql/pgsql.Deployment.yaml

```diff
@@ -51,6 +51,8 @@ spec:
         volumeMounts:
         - mountPath: /data
           name: disk
+        - mountPath: /conf
```

Comment:
Postgres expects the config file in $PGDATA. I don't see immediately how we're telling it to look for the config file in another place. What am I missing?

Reply:
this: https://github.com/sourcegraph/infrastructure/pull/1728

Reply:
Thanks @uwedeportivo!